turbopack/crates/turbopack-node/src/execution_context.rs

```diff
@@ -8,6 +9,7 @@ pub struct ExecutionContext {
     pub project_path: ResolvedVc<FileSystemPath>,
     pub chunking_context: ResolvedVc<Box<dyn ChunkingContext>>,
     pub env: ResolvedVc<Box<dyn ProcessEnv>>,
+    pub encryption_key: RcStr,
```

Comment:
I thought using the `execution_context` that's received by the transforms could be the nicer way to reference the `encryption_key` without having to add a new param.